e2e-tests/testdata/echo_client/js/client.js

```diff
@@ -13,6 +13,13 @@ export function Environment(name) {
     return `https://${name}-slug.encr.app`
 }
 
+/**
+ * PreviewEnv returns a BaseURL for calling the preview environment with the given PR number.
+ */
+export function PreviewEnv(pr) {
+    return Environment(`pr${pr}`)
+}
```

Comment:
Should we add this function to the Go client too?

Reply:
Good idea, done